client/src/components/Applications/AppForm/AppFormUtils.test.js

```diff
@@ -73,16 +70,12 @@ describe('AppFormUtils', () => {
   });
 
   it('handles queue validation on Frontera HPCs for SERIAL apps', () => {
-    /* TODOv3 Add small queue to fixture https://jira.tacc.utexas.edu/browse/TV3-98
     expect(
-
-
-      (smallQueue, helloWorldAppFixture).isValidSync('small')
+      getQueueValidation(smallQueue, serialFronteraApp).isValidSync('small')
```

Comment:
FYI, in v3 a queue has a `minNodeCount` field, so we won't need to look for any special cases anymore. Have a branch working on that, but no jira task I don't think. I'll add one

Reply:
augh, i thought we had already updated that in the original PR to use `minNodeCount` instead of the special function checking for Frontera/normal

Reply:
@rstijerina  good cach